EventManagementSystem/src/main/java/com/eventmanagementsystem/EventManagementSystem/service/BookingServiceImpl.java

```diff
@@ -0,0 +1,87 @@
+package com.eventmanagementsystem.EventManagementSystem.service;
+
+import com.eventmanagementsystem.EventManagementSystem.model.Booking;
+import com.eventmanagementsystem.EventManagementSystem.model.Event;
+import com.eventmanagementsystem.EventManagementSystem.model.User;
+import com.eventmanagementsystem.EventManagementSystem.repository.BookingRepository;
+import com.eventmanagementsystem.EventManagementSystem.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class BookingServiceImpl implements BookingService{
+
+    @Autowired
+    public BookingRepository bookingRepository;
+
+    @Autowired
+    public EventService eventService;
+
+    @Autowired
+    public UserRepository userRepository;
+
+    @Override
+    public void bookEvent(Booking booking) {
+
+        if(userRepository.findByUserId(booking.getUserId())!=null) {
+            if (bookingRepository.checkAlreadyBooked(booking.getEventId(), booking.getUserId()) == null && eventService.getEventByEventId(booking.getEventId()) != null) {
+                Booking newBooking = new Booking();
+                newBooking.setEventId(booking.getEventId());
+                newBooking.setUserId(booking.getUserId());
+                newBooking.setAttendedStatus("none");
```

Comment:
?

Reply:
setting the values to add this to the mongodb collection

Reply:
default attended status set as none